frontend/src/app/testResults/uploads/CsvSchemaDocumentation.test.tsx

```diff
@@ -161,6 +163,34 @@ describe("CsvSchemaDocumentation tests", () => {
       );
       expect(container).toMatchSnapshot();
     });
+    it("has working tabs for navigation", async () => {
+      render(
+        <MemoryRouter initialEntries={["/results/upload/submit/guide"]}>
+          <Routes>
+            <Route
+              path={"/results/upload/submit/guide"}
+              element={
+                <CsvSchemaDocumentation
+                  schemaBuilder={specificSchemaBuilder}
+                  returnUrl={"/results/upload/submit"}
+                />
+              }
+            />
+          </Routes>
+        </MemoryRouter>
+      );
+      const user = userEvent.setup();
+
+      expect(screen.getAllByText("Required for Positives")).toHaveLength(1);
```

Comment:
⚠️ expecting it to have length 1 because we use the `Required for Positives` tag in the explanation of how the tags work at the start of the guide